### DISCUSSION TITLE
File upload validation error instead of warning

### DISCUSSION THREAD

**Discussion by srikanthgurram-17** [wcm-io/io.wcm.handler.media#68] [Q&A] [answered]
@stefanseifert  We would like to show error validation message instead of warning message whenever the asset media is invalid in file upload dialog selection. is it possible to have a hook inside MediaFormatValidate showAlert method for the consumer to customise to a error instead of warning?

**Comment by stefanseifert** [comment]
yes, this would be a good addition. i think we can build in this feature into the media handler Granite UI components, with the option to enable it via a configuration parameter.

**Comment by stefanseifert** [comment, marked as answer]
PR: https://github.com/wcm-io/io.wcm.handler.media/pull/69
